### ISSUE TITLE
In 200% zoom, Keyboard focus is not properly visible on 'Access token' and 'Got Feedback' tab item button.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1374]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator
Username: hcltester200@outlook.com/hcltester800@outlook.com

**Pre-Requisite:**
1.Setting Display Resolution to '1280*768
2.Select Zoom settings to '200%'  for Anaheim dev browser.
3.Refresh the page.

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Navigate to the 'Access token' button using tab key.
4. Observe if keyboard focus is properly visible.

**Actual Result:**
In 200% zoom, Keyboard focus is not properly visible on 'Access token' and 'Got Feedback' tab item button.

**Expected Result:**
In 200% zoom, Keyboard focus should be properly visible on 'Access token' and 'Got Feedback' tab item button.

**User Impact:** 
Visual impairment users will face difficulty if any item on the application or keyboard focus on it is not visible on resizing the page to a 200% zoom level
 
**MAS Reference:**
[Resize Text - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.04.04)

**Comment by ElinorW** [comment]
tracking this here https://github.com/microsoft/fluentui/issues/20664

**Comment by ElinorW** [comment]
No longer an issue as the Feedback button was moved in this PR https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/1455
